public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -255,6 +262,32 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
    * The unchanged pre 7.1 query implementation
    */
   classicQuery(options: any): Observable<DataQueryResponse> {
+    // migrate annotations
+    if (options.targets.some((target: InfluxQuery) => target.fromAnnotations)) {
+      const streams: Array<Observable<DataQueryResponse>> = [];
+
+      for (const target of options.targets) {
+        if (target.hide) {
+          continue;
+        } else {
+          streams.push(
+            new Observable((subscriber) => {
+              this.annotationEvents(options, target)
+                .then((events) => {
+                  return subscriber.next({ data: [toDataFrame(events)] });
+                })
+                .catch((ex) => {
+                  return subscriber.error(new Error(ex));
+                })
+                .finally(() => subscriber.complete());
+            })
+          );
+        }
```

Comment:
How about removing the `continue` statement?
```ts
if (!target.hide) {
    ...
}
```

Reply:
Yes! We actually don't need the hide here, it is handled elsewhere in Influx datasource. I'll remove it.